ui/src/app/services/notebooks-swagger-fetch-clients.ts

```diff
@@ -0,0 +1,105 @@
+/**
+ * This package configures/provides swagger "typescript-fetch" client
```

Comment:
Maybe just reference the other file instead to avoid duplicating this documentation.